src/lib/SX1280Driver/SX1280_hal.cpp

```diff
@@ -86,13 +86,14 @@ void SX1280Hal::init()
 #endif
 
 #ifdef PLATFORM_STM32
+    Serial.println("Config SPI");
     SPI.setMOSI(GPIO_PIN_MOSI);
     SPI.setMISO(GPIO_PIN_MISO);
     SPI.setSCLK(GPIO_PIN_SCK);
     SPI.setBitOrder(MSBFIRST);
     SPI.setDataMode(SPI_MODE0);
     SPI.begin();
-    SPI.setClockDivider(SPI_CLOCK_DIV4); // 72 / 8 = 9 MHz
+    SPI.setClockDivider(SPI_CLOCK_DIV16); // 72 / 8 = 9 MHz
```

Comment:
Change back and fix the comment.

Reply:
comment is actually valid since SPI peripheral clock is McuClock / 2 = 36MHz. So actual SPI clock is 72MHz / 2 / 4 for STM32F103 MCUs.